pkg/types/v1/bootloader.go

```diff
@@ -17,12 +17,11 @@ limitations under the License.
 package v1
 
 type Bootloader interface {
-	Install(rootDir, bootDir, deviceLabel string) (err error)
+	Install(rootDir, bootDir string) (err error)
 	InstallConfig(rootDir, bootDir string) error
 	DoEFIEntries(shimName, efiDir string) error
-	InstallEFI(rootDir, bootDir, efiDir, deviceLabel string) error
```

Comment:
`bootDir` and `efiDir` were set always to the same value and the code is not really prepared to properly use them as different concepts.